.NET/Microsoft.Recognizers.Text.Number/Parsers/BaseCJKNumberParser.cs

```diff
@@ -655,6 +661,12 @@ private double GetIntValue(string intStr)
 
                 hasPreviousDigits = char.IsDigit(intStr[i]);
 
+                // Japanese numbers in the form "一九九九" (1999) must be processed as digit numbers
+                if (Config.CultureInfo.Name == "ja-JP" && !hasPreviousDigits)
```

Comment:
Use a constant for the culture definition.

Reply:
done